ext/opentelemetry-ext-asyncpg/src/opentelemetry/ext/asyncpg/__init__.py

```diff
@@ -58,7 +66,16 @@ def _exception_to_canonical_code(exc: Exception) -> StatusCanonicalCode:
     return StatusCanonicalCode.UNKNOWN
 
 
-def _hydrate_span_from_args(connection, query, parameters) -> dict:
+def _mask_parameters(parameters):
+    serialized = json.dumps(parameters, sort_keys=True)
+    masked = hashlib.sha256(serialized.encode("utf-8"))
+
+    return ":".join((masked.name, masked.hexdigest()))
+
+
+def _hydrate_span_from_args(
```

Comment:
Maybe it makes sense to have this as a method on the class to avoid passing `mask_parameters` like you did with `_do_execute()`